docs/documentation/functions/vsphere/Copy-vSphereRole.md

```diff
@@ -0,0 +1,149 @@
+# Copy-vSphereRole
+
+## Synopsis
+
+Copies a vSphere role.
+
+## Syntax
+
+```powershell
+Copy-vSphereRole [-server] <String> [-user] <String> [-pass] <String> [-sddcDomain] <String> [-sourceRoleName] <String> [-targetRoleName] <String> [-ProgressAction <ActionPreference>] [<CommonParameters>]
+```
+
+## Description
+
+The `Copy-vSphereRole` cmdlet copies a role in vCenter Server.
+
+The cmdlet connects to SDDC Manager using the -server, -user, and -password values:
+
+- Validates that network connectivity and authentication is possible to SDDC Manager
+- Validates that network connectivity and authentication is possible to vCenter Server
+- Requests the set of privileges assigned to a specific role in vCenter Server and creates a new role with the same privileges.
+
+## Examples
+
+### Example 1
+
+```powershell
+Copy-vSphereRole -server sfo-vcf01.sfo.rainpole.io -user administrator@vsphere.local -pass VMw@re1! -sddcDomain sfo-m01 -sourceRoleName "Admin" -targetRoleName "Aria Automation Orchestrator to vSphere Integration"
+```
+
+This example requests the set of privileges assigned the Administrator role in vCenter Server and creates a new role named "Aria Automation Orchestrator to vSphere Integration" with the same privileges.
+
+## Parameters
+
+### -server
+
+The fully qualified domain name of the SDDC Manager.
+
+```yaml
+Type: String
+Parameter Sets: (All)
+Aliases:
+
+Required: True
+Position: 1
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -user
+
+The username to authenticate to the SDDC Manager.
+
+```yaml
+Type: String
+Parameter Sets: (All)
+Aliases:
+
+Required: True
+Position: 2
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -pass
+
+The password to authenticate to the SDDC Manager.
+
+```yaml
+Type: String
+Parameter Sets: (All)
+Aliases:
+
+Required: True
+Position: 3
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -sddcDomain
+
+The workload domain in SDDC Manager.
+
+```yaml
+Type: String
+Parameter Sets: (All)
+Aliases:
+
+Required: True
+Position: 4
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -sourceRoleName
+
+The name of the role to copy.
+
+```yaml
+Type: String
+Parameter Sets: (All)
+Aliases:
+
+Required: True
+Position: 5
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -targetRoleName
+
+The name of the new role.
+
+```yaml
+Type: String
+Parameter Sets: (All)
+Aliases:
+
+Required: True
+Position: 6
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -ProgressAction
+
+Progress Action
+
+```yaml
+Type: ActionPreference
+Parameter Sets: (All)
+Aliases: proga

```

Comment:
Is this Alias supposed to be proga or progress?

Reply:
Thanks! This is the output from the PlatyPS module that generates the documentation from the function.